assets/js/wcmp-admin.js

```diff
@@ -867,3 +848,36 @@ jQuery(function($) {
   }
 });
 
+/**
+ * Object.assign() polyfill.
+ */
+if (typeof Object.assign !== 'function') {
+  // Must be writable: true, enumerable: false, configurable: true
+  Object.defineProperty(Object, 'assign', {
+    value: function assign(target, varArgs) { // .length of function is 2
```

Comment:
Als de code geminified is, zal `//` voor problemen zorgen 

```suggestion
/* eslint-disable-next-line max-lines-per-function */
```
```suggestion
    value: function assign(target, varArgs) { /* .length of function is 2*/
```